content/rs/administering/database-operations/slave-ha.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: High Availability for Slave Shards
+description: 
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+When Replication is enabled and master shard fails, the slave shard is automatically promoted to a master shard 
+to maintain data availability. This creates a single point of failure until a new slave shard is manually created.
```

Comment:
What about the situation where the slave shard fails? It's also relevant here. In such case we also have a single point of failure, and then a new slave shard is created and the data is replicated from the master to the new slave shard

Reply:
See https://github.com/RedisLabs/redislabs-docs/pull/108/commits/86ab4797cf30795976cc988463a416b0d5f04b03